### ISSUE TITLE
PDE uses a different stack then the regular stack for its views

### ISSUE THREAD

**Issue by vogella** [eclipse-pde/eclipse.pde#802]
Last week I gave an introduction to a group of new PDE users and the main complaint they had was that we are using two stacks in case PDE perspective is used.

![image](https://github.com/eclipse-pde/eclipse.pde/assets/139910/ca9b4219-291f-45a6-88e1-c09b395e0705)

To reproduce, reset your PDE perspective and open the Console.

Still very annoying. Git Tooling seems to have no issue using the "regular" bottom stack. 

IIRC this was reported in the past but I think in Bugzilla.

**Comment by Dinesh0723** [comment]
@vogella , refer the attached image,
![2024-04-11_13h59_43](https://github.com/eclipse-pde/eclipse.pde/assets/130960675/67c70e76-9204-465e-8539-5470d7ca5d56).

Are there any other views that need to be opened in the same stack?

**Comment by vogella** [comment]
@Dinesh0723 did you fix something? What if you reset the perspective?

**Comment by Dinesh0723** [comment]
> @Dinesh0723 did you fix something? What if you reset the perspective?

Yeah, I have added a placeholder for respective views in the above screenshot.

If we do reset perspective, it will show only the default views (Problems view, Target Platform State view) in bottom stack..

**Comment by vogella** [comment]
Looks good to me. Thanks for the fix. Issue can be closed in case your fix is merged.

**Comment by Dinesh0723** [comment]
Sure. Thank you.

**Comment by vogella** [comment]
Thank you for fixing

**Comment by HannesWell** [comment]
> Looks good to me. Thanks for the fix. Issue can be closed in case your fix is merged.

@Dinesh0723 can you please link your PR here?

**Comment by vogella** [comment]
Fixed via https://review.gerrithub.io/c/eclipse-egit/egit/+/1245418
